turbinia/config/turbinia_config_tmpl.py

```diff
@@ -250,4 +250,4 @@
 
 # Prometheus listen address and port
 PROMETHEUS_ADDR = '0.0.0.0'
-PROMETHEUS_PORT = 8000
+PROMETHEUS_PORT = 9200
```

Comment:
If we change this we need to update the prometheus instance in the lab! Can you take care of that when you merge this PR?